packages/@okta/vuepress-site/docs/guides/pwd-optional-new-sign-up-email/main/react/integrationsteps.md

```diff
@@ -0,0 +1,186 @@
+
+```javascript
+const inputValues = {
+  firstName: "John",
+  lastName: "Doe",
+  email: "johndoe@email.com",
+ };
+```
+
+Send this new object to `OktaAuth.idx.proceed()`.
+
+```javascript
+setProcessing(true);
+const newTransaction = await oktaAuth.idx.proceed(inputValues);
+setTransaction(newTransaction);
+```
+
+>**Note**: You can also start the registration flow in a single step by passing the account details in `OktaAuth.idx.register()`. See [`idx.register`](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#idxregister) in the GitHub docs or [Node Express's](/docs/guides/pwd-optional-new-sign-up-email/nodeexpress/main/#_3-the-user-submits-their-new-account-details) version of this guide to learn more.
+
+### 4. Identity Engine requests new email verification
+
+`OktaAuth.idx.proceed()` returns an `IdxTransaction` object with a `status` of `PENDING`, indicating that the user needs to verify their identity with their email.
+
+```json
+{
+  status: "PENDING",
+  nextStep: {
+    name: "enroll-authenticator",
+    type: "email",
+    authenticator: {
+      type: "email",
+      key: "okta_email",
+      displayName: "Email",
+    },
+  },
+}
+```
+
+Build the logic that handles this response and sends the user to a dialog where they enter the OTP.
+
+### 5. The user verifies their identity with the new email
+
+The user opens the email sent by Identity Engine and copies the one-time passcode (OTP) to your app. Create a dialog to allow the user to enter and submit the OTP.
+
+<div class="half border">
+
+![Screenshot showing a page with a OTP input field.](/img/pwd-optional/pwd-optional-change-email-my-account-js-react-verify-email.png)
+
+</div>
+
+When the user submits the OTP, create an object with a `verificationCode` property set to the OTP entered by the user.
+
+```json
+const inputValues = {
+    "verificationCode":"197277"
+ };
+```
+
+Call `OktaAuth.idx.proceed()` passing in the new object.
+
+```javascript
+    const newTransaction = await oktaAuth.idx.proceed(inputValues);
+    setTransaction(newTransaction);
+    setInputValues({});
+    setProcessing(false);
+```
+
+### 6. Your app displays the remaining optional authenticators
+
+After the user verifies their identity using the email authenticator, `OktaAuth.idx.proceed()` returns an `IdxTransaction` object indicating that the user has the option to enroll in additional authenticators. The `IdxTransaction.nextStep.canSkip` property is set `true`, which indicates the remaining authenticators are optional.
+
+```json
+{
+  status: "PENDING",
+  nextStep: {
+    name: "select-authenticator-enroll",
+    options: [
+      {
+        label: "Okta Verify",
+        value: "okta_verify",
+      },
+      {
+        label: "Password",
+        value: "okta_password",
+      },
+    ],
+    canSkip: true,
+  },
+}
+```
+
+Display a page that lists the remaining authenticators and allows the user to skip registering any more of them.
+
+<div class="half border">
```

Comment:
Same note as earlier regarding the CSS border

Reply:
See previous comment. Images have been fixed in Figma.